controller/src/main.cpp

```diff
@@ -65,7 +65,7 @@ static GuiStatus gui_status = GuiStatus_init_zero;
 //
 // Uncomment this line to get started:
 //
-//   #define NO_GUI_DEV_MODE
+#define NO_GUI_DEV_MODE
```

Comment:
Is this intended to be part of this PR?

Reply:
Ugh, can't get rid of this  soon enough.  Thanks.